dracut.sh

```diff
@@ -450,6 +452,7 @@ rearrange_params() {
             --long keep \
             --long printsize \
             --long regenerate-all \
+            --long no-regenerate-all \
```

Comment:
This is intended to be a "hidden" option that will be called during the "regenerate all" process, as it does not make much sense to a regular user because the default behavior is not to regenerate all initrds.